### PR TITLE
Remove scan type from reports

### PR DESCRIPTION
**Pull request by Nicoolai**
### Describe the Pull Request

```
Currently for zipped source files, the upload api doesn't return what type of zip it is. 
This needs to be explicitly set or scan should be able to happen without specifying the type of zip, ie andzip or ioszip .

I have changed the report calls to try all static_db's in search for the report. It then uses whatever returns a result, making it able to automatically figure out the source type.
```

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`.
- [ ] Tested Working on Linux, Mac, Windows, and Docker
- [ ] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### Additional Comments (if any)

```
I have also updated tests and API docs.
```

### CODE REVIEW

File: StaticAnalyzer/tests.py [hunk 1]
```diff
@@ -52,18 +52,18 @@ def static_analysis_test():
         logger.info('Running PDF Generation Test')
         if platform.system() in ['Darwin', 'Linux']:
             pdfs = [
-                '/PDF/?md5=3a552566097a8de588b8184b059b0158&type=APK',
```
Comment (Nicoolai) [inline StaticAnalyzer/tests.py]: I am personally a bit uncertain about these changes.  
Not entirely sure if this "PDF Generation Test" is something that should be changed.  
So please, add any feedback regarding this.

Comment (ajinabraham) [reply StaticAnalyzer/tests.py]: Looks good to me. We want to run PDF generation tests for all kind of source formats we support.

**Comment by ajinabraham** [comment]
Code looks good to me. Testing this out. Thanks for the PR

**Comment by ajinabraham** [comment]
Testing went good. great work!

**Pull request merged**